src/server/resources/queries/vehicle/VEHICLE_NEW_CONTINGENCY_MOT.sql

```diff
@@ -0,0 +1,31 @@
+select veh.registration, date_format(mtc.issued_date,'%Y%m%d') as issued_date, mtc.number
+from vehicle veh, model_detail md,
+  (select max(id) as id, vehicle_id  from mot_test_current
```

Comment:
I include this as a sub query in a different way in my alternative query - since the query appears to only need 50 rows based on the limit this seems a bit of a heavy way to get 50 rows.